ios/ReactNativeUsabilla/UsabillaInterface.swift

```diff
@@ -83,6 +83,25 @@ class UsabillaInterface: NSObject {
     func takeScreenshot(view: UIView) -> UIImage {
         return Usabilla.takeScreenshot(view)!
     }
+    
+    @objc(dismiss)
+    func dismiss() {
+        let _ = Usabilla.dismiss()
```

Comment:
Does it need to be assigned a value to a variable?
If so should we return it?

Reply:
To avoid warning we use _ operator.
As assigning it to unused constant or variable cause warning.